viam-agent.service

```diff
@@ -11,6 +11,7 @@ RestartSec=5
 User=root
 TimeoutSec=240
 ExecStart=/opt/viam/bin/viam-agent --config /etc/viam.json
+KillMode=mixed
```

Comment:
Please, add a comment that this file must be kept in sync with the preinstall script. Thank you